spot-client/src/index.js

```diff
@@ -58,7 +59,8 @@ const store = createStore(
         spotTv: {
 
             // Will get overridden on Spot-Remote by Spot-TV updates.
-            electron: isElectron()
+            electron: isElectron(),
+            volumeControlSupported: getBoolean(startParams.volumeControlSupported)
```

Comment:
what happens on page reload? does this need to persist?

Reply:
Good point, added. Also, implemented in a way that the URL param will always overwrite the persisted value, if present.